rbi/annotations/sidekiq.rbi

```diff
@@ -75,6 +75,9 @@ module Sidekiq::Job::ClassMethods
 
   sig { params(interval: T.untyped, args: T.untyped).returns(String) }
   def perform_in(interval, *args); end
+
+  sig { params(args: T.untyped, kwargs: T.untyped).returns(String) }
+  def perform_bulk(*args, **kwargs); end
```

Comment:
The `perform_bulk` method in `Job` indeed accepts [**kwargs](https://github.com/sidekiq/sidekiq/blob/352df475fceec348a543216f2f1fdddc047e8cc6/lib/sidekiq/job.rb#L315), but under the hood it invokes `Setter#perform_bulk`, which only accepts [batch_size](https://github.com/sidekiq/sidekiq/blob/352df475fceec348a543216f2f1fdddc047e8cc6/lib/sidekiq/job.rb#L240).

Can you actually invoke this method with any keyword arguments that aren't `batch_size`? If not, maybe it's worth making it explicit.

Reply:
This is actually telling Sorbet to accept keyword arguments and that each one of them will be a `{ batch_size: Integer }` shape. To get it to accept only `batch_size`, you need to do

```ruby
sig { params(args: T.untyped, batch_size: Integer).returns(String) }
def perform_bulk(*args, batch_size: 1000); end
```